civcraftConfigs/alephconfig.yml

```diff
@@ -126,7 +126,7 @@ monster:
    OPskeletons:
      identifier: firewatchskelly
      type: SKELETON
-     name: RabidFireWatch
+     name: §kFive §4RabidFireWatch §kFive 
```

Comment:
yepp were tryin to have names generate 5 random letter and numbers plus coloring the name of the mob for testing